x/auth/vesting/types/vesting_account.go

```diff
@@ -820,8 +821,10 @@ func (va *ClawbackVestingAccount) computeClawback(clawbackTime int64) sdk.Coins
 	totalUnvested := sdk.NewCoins()
 	unvestedIdx := 0
 	for i, period := range va.VestingPeriods {
+		// this period vests at time t, if this occurred before clawback time,
+		// then its already vested.
 		t += period.Length
```

Comment:
`t` is not descriptive naming of the variable and what it is used for

Reply:
`t` is the time of each vesting event. Since `time` is also an important package name, I try to avoid it as a variable name.

Reply:
That being said, upon looking at uses of `t` I noticed a bug, that might have been more evident if the variable were named `vestTime`.  Please see issue Agoric/agoric-sdk#5134 for the bug description and PR agoric-labs/cosmos-sdk#207 for the fix. I suggest porting this PR since this code has the same issue.